### PR TITLE
Full Transit Path

### PR DESCRIPTION
**Pull request by awiss**
I just learned
-
you can use markdown here

### CODE REVIEW

**Comment by yefim** [comment]
@awiss LGTM

**Comment by adelq** [comment]
**Indeed you can**

File: server/transit.py [hunk 1]
```diff
@@ -68,12 +67,21 @@ def get_data():
...
+
+      path = reduce(add_path_points, stops[fromStop['order']:toStop['order']+1], [])
       possible_routes.append({
         'route_name': route_name,
         'walkingDistanceBefore': minFrom,
-        'path': stops[fromStop['order']:toStop['order']+1],
+        'path': path,
         'walkingDistanceAfter': minTo
-        })
+        });
```
Comment (adelq) [inline server/transit.py]: Semicolon?

**Comment by adelq** [comment]
LGTM after nit

File: server/transit.py (hunk 1 above)
Comment (awiss) [inline server/transit.py]: NODE.JS 4 LYFE
-

Comment (adelq) [inline server/transit.py]: [penn-sdk-node](pennlabs/penn-sdk-node) is beckoning for you

Comment (adelq) [inline server/transit.py]: Actually, penn-sdk-clojure is

**Comment by adelq** [comment]
Can you deploy this since you have the Google Maps key?

**Pull request merged**